.vscode/tasks.json

```diff
@@ -0,0 +1,141 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Aerie UI",
+      "type": "shell",
+      "command": "nvm use && npm run dev",
+      "detail": "Task to run just the UI for development.",
```

Comment:
Maybe "Run the Aerie UI in development mode"?

Reply:
Though I guess you have `Task to` prefixing all of these details so would be good to keep it consistent..

Reply:
Another thing we could do here is `Aerie: <Task Name>` to keep it nice and namespaced within tasks?

Reply:
yeah! i like that idea. but what would the namespaces be?

Reply:
I was picturing all of these tasks being named `Aerie: <Task Name>`, like `Aerie: Run Development UI`

Reply:
Ah, so if it's all namespaced `Aerie`, then I think it'd be more useful to easily see the actual task name because the task bar takes up a bit of horizontal space on the screen.

![Screenshot 2024-04-02 at 12 23 19 PM](https://github.com/NASA-AMMOS/aerie-ui/assets/5290214/dcccf84b-9b10-4765-869d-6e3f4b752634)

![Screenshot 2024-04-02 at 12 22 27 PM](https://github.com/NASA-AMMOS/aerie-ui/assets/5290214/88a2e0fc-cc8d-405f-a652-8b4e4cc12cc1)

Reply:
Aha good point!